<launch>	
	<!--load config file for perception_pipeline parameters-->
	<rosparam command="load" file="$(find perception_pipeline)/config/config.yaml" />

	<!-- run perception_yolo node node -->
	<node name="perception_yolo" pkg="perception_pipeline" type="perception_yolo_new.py" output="screen" />

</launch>
